nemo/collections/asr/models/classification_models.py

```diff
@@ -396,11 +397,13 @@ def multi_validation_epoch_end(self, outputs, dataloader_idx: int = 0):
         self._accuracy.total_counts_k = total_counts
         topk_scores = self._accuracy.compute()
 
-        tensorboard_log = {'val_loss': val_loss_mean}
+        self.log('val_loss', val_loss_mean)
```

Comment:
For multi validation and multi test, return the dictionary as before. Internally it will prepend dataset names and then call self.log_dict()